embassy-stm32/src/usb/otg.rs

```diff
@@ -26,7 +27,6 @@ impl<T: Instance> interrupt::typelevel::Handler<T::Interrupt> for InterruptHandl
     unsafe fn on_interrupt() {
         let r = T::regs();
         let state = T::state();
-
```

Comment:
I'd not remove that empty line but that's really just a nitpick

Reply:
fixed.

Reply:
Almost, there is now a newline behind instead of before the function call :D
It was a nit as usually there is an empty line between variable population and and their usage but I don't think there is a styleguide for this for I would have not changed it here.

Reply:
please avoid nitpicking style stuff. we don´t have a styleguide beyond "use rustfmt" (which CI enforces).

Reply:
Commented again as `rustfmt` complains now but I'll refrain from any style nitpicks now. I was working in projects with some strict guidelines so consider my initial comment muscle memory.

Reply:
OK, ran the fmt. Should be resolved.